backend/src/Designer/Filters/IO/IoExceptionFilterAttribute.cs

```diff
@@ -0,0 +1,32 @@
+using System.IO;
+using System.Net;
+using Altinn.Studio.DataModeling.Converter.Csharp;
+using Altinn.Studio.DataModeling.Converter.Json;
+using Altinn.Studio.DataModeling.Converter.Metadata;
+using Altinn.Studio.DataModeling.Converter.Xml;
+using Altinn.Studio.Designer.Filters.DataModeling;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Controllers;
+using Microsoft.AspNetCore.Mvc.Filters;
+
+namespace Altinn.Studio.Designer.Filters.IO;
+
+public class IoExceptionFilterAttribute : ExceptionFilterAttribute
+{
+    public override void OnException(ExceptionContext context)
+    {
+        base.OnException(context);
+
+        if (context.ActionDescriptor is not ControllerActionDescriptor)
+        {
+            return;
+        }
+
+        // TODO: Implement custom IO exceptions
+        if (context.Exception is FileNotFoundException)
+        {
+            // TODO: Implement custom IO exceptions Error Codes
+            context.Result = new ObjectResult(ProblemDetailsUtils.GenerateProblemDetails(context.Exception, "File not found", HttpStatusCode.InternalServerError)) { StatusCode = (int)HttpStatusCode.InternalServerError };
```

Comment:
Not sure if InternalServerError is the best choice here. Resource that we're trying to get does not exist on server. Is 404 better option here?

Reply:
Ah sorry, that was just i blip - didn't see what statuscode i copied 😬 Det gikk fort i svingene, as we say in Norway 😅